main.rb

```diff
@@ -0,0 +1,84 @@
+require_relative './app'
+require './app'
+
+# * Select 4
+def create_a_person
+  puts 'Do you want to create (1) a Student or (2) a Teacher?'
+  selected = gets.chomp.to_i
+  print 'Age: '
+  age = gets.chomp
+  print 'Name: '
+  name = gets.chomp
+  case selected
+  when 1
+    print 'Has Parent Permission (Y/N): '
+    parent_permission = gets.chomp
+    create_student(age, name, parent_permission, 'nil')
+  when 2
+    print 'Specialization: '
+    specialization = gets.chomp
+    create_teacher(age, name, specialization)
+  end
+  main
+end
+
+# rubocop:disable Style/GlobalVars
```

Comment:
- [ ] Your app works fine but I noticed that you disabled **Rubocop** a lot is not a good practice. Please remove all the **Rubocp disable** statements in your codebase and try to fix all the errors you are getting. Because ` Rubocop` helps developers to write clean Ruby code and adapt best practices when writing the code.  :+1: